src/apps/solid-lizard/common/solid-lizard.voting-rewards.contract-position-fetcher.ts

```diff
@@ -0,0 +1,130 @@
+import { Inject, NotImplementedException } from '@nestjs/common';
+import { BigNumber, Contract } from 'ethers/lib/ethers';
+import _, { range, sumBy } from 'lodash';
+
+import { APP_TOOLKIT, IAppToolkit } from '~app-toolkit/app-toolkit.interface';
+import { drillBalance } from '~app-toolkit/helpers/drill-balance.helper';
+import { DefaultDataProps } from '~position/display.interface';
+import { ContractPositionBalance, RawContractPositionBalance } from '~position/position-balance.interface';
+import { MetaType } from '~position/position.interface';
+import { ContractPositionTemplatePositionFetcher } from '~position/template/contract-position.template.position-fetcher';
+import { GetTokenDefinitionsParams } from '~position/template/contract-position.template.types';
+
+import { SolidLizardDefinitionsResolver } from '../common/solid-lizard.definitions-resolver';
+import { SolidLizardBribe, SolidLizardContractFactory } from '../contracts';
+
+export abstract class VotingRewardsContractPositionFetcher<
+  T extends Contract,
+> extends ContractPositionTemplatePositionFetcher<T> {
```

Comment:
When you override `async getBalances`, the class has to extend `CustomContractPositionTemplatePositionFetcher`. I'll do the update on the velodrome app integration

Recently found out some balances are missing when extending `ContractPositionTemplatePositionFetcher` and the class is overriding getBalance